### PR TITLE
gird is more interative

### PR DESCRIPTION
**Pull request by Hoobachube**

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -10,4 +10,7 @@
     height: 100px;
     width: 100px;
     float: left;
-}
```
Comment (alexw1983) [inline main.css:13]: This looks good. Ome small point is that it isn't formatted. Try and install a css formatted from the vs code extensions. There is no functional difference it just makes it easier to read.

File: main.js [hunk 2]
```diff
@@ -0,0 +1 @@
+alert("Hi");
```
Comment (alexw1983) [inline main.js]: Alerts are often considered annoying. Change this to console.log

**Review by alexw1983 — APPROVED** [review]
Good stuff. Some minor points

File: main.js (hunk 2 above)
Comment (Hoobachube) [reply main.js]: I thought I deleted that! That was only in there to check that I had set up the js correctly for the next stage

Comment (alexw1983) [reply main.js]: Yeah I figured. This is why we do PRs :)

File: main.css (hunk 1 above)
Comment (Hoobachube) [reply main.css:13]: This isn't on my version.. Why is that? I know you talked about this before but I don't know why its on the PR but not on my version

Comment (alexw1983) [reply main.css:13]: Check your commits are in sync with the remote. Do git log --oneline on your machine  then check the commits on this PR. If they're not the same then  do a push

**Pull request merged**